client/client.go

```diff
@@ -61,6 +61,17 @@ var (
 	defaultClient Client
 )
 
+// SetLogger sets the global logger for the Dapr client.
+// By default log.New(os.Stdout, "", 0) is used.
+// To disable logging, set the logger to nil.
+func SetLogger(l *log.Logger) {
+	if l == nil {
+		l = log.New(io.Discard, "", 0)
```

Comment:
It is not what the comment says.

Reply:
what do you mean?

Reply:
I think the comment may come across as slightly confusing

Reply:
added your suggestion.